### DISCUSSION TITLE
Cloudflare warp in Sandbox

### DISCUSSION THREAD

**Discussion by testmmo** [HotCakeX/Harden-Windows-Security#267] [Q&A]
Has anyone gotten cloudflare warp to work in a sandbox securely? mullvad works. 

I can get it to work if connect from the host. But I rather keep ISP IP for host work then use CF warp for malware analysis capturing dns requests that are evasive in normal envs. 

Trying to setup an WSB config script so auto installs and does some basic tool setup. This is error, but some solutions people say is disable firewall etc. 

![image](https://github.com/HotCakeX/Harden-Windows-Security/assets/26190306/79e2d354-b80e-41fc-9c39-ffca1f034a22)

**Comment by HotCakeX** [comment]
Hi,
Run this as Administrator after installing Warp in Windows Sandbox

```
C:\Program Files\Cloudflare\Cloudflare WARP\warp-svc.exe
```

**Comment by testmmo** [comment]
Thanks works 🥇
